src/features/people/popularPeopleSlice.js

```diff
@@ -0,0 +1,36 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const popularPeopleSlice = createSlice({
+  name: "popularPeople",
+  initialState: {
+    page: 1,
+    totalPages: 500,
```

Comment:
Nie wiem czy te wartości są tutaj potrzebne, ale jeśli tak to wydaje mi się że nie powinny być wpisywane na sztywno. Api zwraca informację o tym ile jest stron dla danego zapytania i to nie zawsze musi być 500 (nie sprawdzałam tego, ale nawet jeśli teraz jest zawsze 500 to w przyszłości to się może zmienić albo można się kiedyś przepiąć na inne api tak już myśląc życiowo rozwojowo). No i przy wyszukiwaniu filmów i ludzi na pewno nie będzie to sztywna ilość stron.

Reply:
Na początku miałem w ogóle wątpliwości czy to będzie potrzebne ale po przemyśleniu wydaje mi się, że może faktycznie tak. Zgadzam się z uwagą @katkowa, inicjalny stan to powinno być 0 albo null bo dopóki nie ma połączenia z API to ten stan jest niewiadomy, po pobraniu to API ustawi odpowiednie wartości. Ewentualnie to page: 1 może zostać bo w sumie to będzie domyślna strona.